### ISSUE TITLE
Don't install

### ISSUE THREAD

**Issue by EQuimper** [nathantreid/meteor-css-modules#47]
Hey, I was trying to install your packages and this morning I get this message in my terminal 

```
meteor add nathantreid:css-modules
 => Errors while adding packages:

While checking for nathantreid:css-modules@1.3.0:
error: No compatible binary build found for this package. Contact
the package author and ask them to publish it for your platform.
```

I have the latest meteor versions 1.3.3.1

**Comment by Spiralis** [comment]
There seems to be a lot of difficulties using this release with an evolving meteor. The meteor react toolbox example referred to is running as is, but updating meteor seems to invalidate the solution. 

I am on meteor 1.3.3.1 and it also fails to install with the message as @EQuimper refers to above.

**Comment by akanix42** [comment]
This is because I'm in the process of releasing version 1.3.0, but the Meteor OS X and linux build servers aren't working properly when they try to release my package. This has happened with basically every CSS Modules package release I've done - it'll not work for a while but I'll keep trying and eventually it'll go through. In the meantime, if you try to install it from Atmosphere on Windows or OS X you get an error.

**Comment by akanix42** [comment]
I just released version 2.0.2. As of 2.0.0, the node-sass, stylus, and sugarss npm packages are no longer bundled with the plugin, which means I no longer have to use the Meteor build servers and can release updates much more quickly/easily. See [https://github.com/nathantreid/meteor-css-modules#breaking-changes-in-200] for more details.

It also means less bloat for the end user. :smile: 

I also updated the react toolbox demo app.
